05.marketplace/03.sell/01.add-item/docs.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Add Your Item
+taxonomy:
+    category: docs
+---
+
+Once you've created an avatar, wearable or entity, you have the option to sell it in the Marketplace. This guide walks you through the steps to upload it to the Marketplace so that you can start the process of having it reviewed and published. 
+
++ [Add Your 3D Model](#add-your-3d-model)
+  + [Add Your FBX File](#add-your-fbx-file)
+  + [Generate Your Item's JSON File](#generate-your-items-json-file)
+
++ [Add Your Avatar](#add-your-avatar)
+  + [Add Your Avatar's Files](#add-your-avatars-files)
+  + [Find Your Avatar's URL](#find-your-avatars-url)
+
++ [Upload Your Environment](#upload-your-environment)
+
+
+
+## Add Your 3D Model
+
+Once you've created your 3D model, you need to add the item and its assets in the Marketplace.
+
+### Add Your FBX File
+
+1. Export your file as an FBX file from your 3D modeling program. Do not export it as a default file type such as .mb or .blend. 
+2. If supported by your 3D modeling software, embed your textures into your FBX file to keep the process simpler.
+3. As a practice and to catch any errors, re-import your FBX back into a new scene and check that it still looks right. Look for things like missing textures or normals, and smoothing issues. You may need to tweak your export settings if the re-imported model does not look right.
+4. If your model could be re-imported without any issues, it is now ready to be uploaded to High Fidelity. Go to [https://highfidelity.com/marketplace/items/new](https://highfidelity.com/marketplace/items/new) to create a new item. If you're not logged in, you will be prompted to do so. Create a new Marketplace item or edit the one you have already created.
```

Comment:
Put the second part in a section called "Add Your 3D Model Assets" and use the steps down below for avatar creation for consistency.

Reply:
Done